libs/widget-lib/src/flexibleConfig.ts

```diff
@@ -33,5 +33,5 @@ export function isPerTradeTypeConfig<T>(config: FlexibleConfig<T>): config is Pe
 export function isPerNetworkConfig<T>(config: FlexibleConfig<T>): config is PerNetworkConfig<T> {
   if (typeof config !== 'object') return false
 
-  return Object.keys(config as object).every((key) => key in SupportedChainId)
+  return Object.keys(config as object).every((key) => typeof key === 'number')
```

Comment:
I think it's ok, we shouldn't hardly link config to the Enum because we can add a support for a new Network and it will add more troubles for widget configs.